apps/omg_watcher/lib/omg_watcher/exit_processor.ex

```diff
@@ -310,9 +310,16 @@ defmodule OMG.Watcher.ExitProcessor do
     {:ok, exit_contract_statuses} = Eth.RootChain.get_standard_exit_structs(get_in(exits, [Access.all(), :exit_id]))
 
     exit_maps =
-      Enum.map(exits, fn exit_event ->
-        put_timestamp_and_sft(exit_event, state.min_exit_period_seconds, state.child_block_interval)
-      end)
+      exits
+      |> Task.async_stream(
+        fn exit_event ->
+          put_timestamp_and_sft(exit_event, state.min_exit_period_seconds, state.child_block_interval)
+        end,
+        timeout: 50_000,
+        on_timeout: :kill_task,
```

Comment:
are we sure we don't want to exit the process?

Reply:
okay, and I refresh the page and is changed to exit 😅

Reply:
no :) you we're 32seconds too quick